app/src/main/res/values/strings.xml

```diff
@@ -551,8 +551,9 @@
     <string name="low_quality_smaller">Low quality (smaller)</string>
     <string name="dont_show">Don\'t show</string>
     <!-- App update notification -->
-    <string name="app_update_notification_content_title">NewPipe update is available!</string>
-    <string name="app_update_notification_content_text">Tap to download</string>
+    <string name="app_update_unavailable_toast">You are running the latest version of NewPipe.</string>
+    <string name="app_update_available_notification_title">NewPipe update is available!</string>
+    <string name="app_update_available_notification_text">Tap to download %s</string>
```

Comment:
Since you added `%s` to this string, I think you should remove all of the copies in other languages so that they are retranslated with the `%s` in the correct position.